app/assets/javascripts/modules/track-briefing.js

```diff
@@ -0,0 +1,34 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function TrackBriefing () {}
+
+  TrackBriefing.prototype.start = function($element) {
+    var scope = $element[0]
+
+    this.trackLinks(scope)
+  }
+
+  TrackBriefing.prototype.trackLinks = function(scope) {
+    var links = scope.getElementsByTagName('a')
+
+    for (var i = 0; i < links.length; i++) {
+      var link = links[i]
+      link.addEventListener('click', this.sendLinkClickEvent)
+    }
+  }
+
+  TrackBriefing.prototype.sendLinkClickEvent = function(event) {
+    GOVUK.analytics.trackEvent(
+      'Briefings page',
```

Comment:
If we generalise this function - does this need be specific about the particular page? Will the event will be assigned with a url of the page it was sent from?

Reply:
It will be assigned with the URL of the page but we should probably change the category too